### PR TITLE
DFPL-1870 - Fix UI test adminMangesHearings_test.js

### PR DESCRIPTION
**Pull request by GauravChawlaMOJ**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-1870

### Change description ###
Fix adminMangesHearings_test.js

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Braimah101 — DISMISSED** [review]

File: e2e/tests/adminManagesOrdersAmendment_test.js [hunk 1]
```diff
@@ -45,7 +46,7 @@ async function setupScenario(I, data = caseData) {
   await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
 }
 
-Scenario('Amend generated order in general case', async ({ I, caseViewPage, manageOrdersEventPage }) => {
+Scenario('@nightlyOnly Amend generated order in general case @nightlyOnly', async ({ I, caseViewPage, manageOrdersEventPage }) => {
```
Comment (GauravChawlaMOJ) [inline e2e/tests/adminManagesOrdersAmendment_test.js:49]: One one @nightly-only should be sufficient

File: package.json [hunk 2]
```diff
@@ -3,12 +3,11 @@
   "version": "1.0.0",
```
Comment (GauravChawlaMOJ) [inline package.json:3]: Can we please revert all changes in package.json? Some seem to be out-of-date,others seem to be helpful for tests to run locally only.

File: e2e/tests/adminManagesHearingsMissingPreviousVenue_test.js
Comment (prabhamuthu15) [inline e2e/tests/adminManagesHearingsMissingPreviousVenue_test.js:1]: This test is not part of the ticket . the changes are incomplete  can rollback these changes

File: e2e/tests/adminManagesOrders_test.js
Comment (prabhamuthu15) [inline e2e/tests/adminManagesOrders_test.js:1]: This test is not part of the ticket . the changes are incomplete can rollback these changes

**Pull request merged**